addons/mail/static/src/owl/components/activity/activity.xml

```diff
@@ -0,0 +1,147 @@
+            <div class="o_Activity">
+                <div class="o_Activity_sidebar">
+                    <div class="o_Activity_user">
+                        <img class="o_Activity_userAvatar rounded-circle"
+                             t-att-class="{
+                                o_Message_authorRedirect: hasAuthorRedirect,
+                                o_redirect: hasAuthorRedirect
+                             }"
+                             t-att-src="assignedAvatarSource"
+                             t-att-alt="storeProps.activity.userDisplayName"/>
+                        <i class="o_Activity_userAvatar_icon fa" t-att-class="iconClasses"/>
+                    </div>
+                </div>
+                <div class="o_Activity_core">
+                    <div class="o_Activity_info">
+                        <span class="o_Activity_delay"
+                              t-att-class="'o_Activity_color_'+storeProps.activity.activityState">
+                            <t t-esc="delayLabel"/>
+                        </span>
+                        :
+                        <t t-if="storeProps.activity.summary">
+                            <span class="o_Activity_summary">
+                                &#8220;<t t-esc="storeProps.activity.summary"/>&#8221;
+                            </span>
+                        </t>
+                        <t t-else="">
+                            <span class="o_Activity_type">
+                                <t t-esc="storeProps.activity.activityTypeDisplayName"/>
+                            </span>
+                        </t>
+                        <span class="o_Activity_userName">
+                            <em> for </em>
+                            <t t-esc="storeProps.activity.userDisplayName" />
+                        </span>
+                        <a class="o_Activity_detailsButton btn btn-link text-muted collapsed"
+                           t-on-click="_toggleDetailsVisibility"
+                           role="button">
+                            <i class="fa fa-info-circle" role="img" aria-label="Info" title="Info"/>
+                        </a>
+                    </div>
+
+                    <div class="o_Activity_details" t-if="state.isDetailsVisible">
+                        <dl class="dl-horizontal">
+                            <dt>Activity type</dt>
+                            <dd class="o_Activity_detailsType">
+                                <t t-esc="storeProps.activity.activityTypeDisplayName"/>
+                            </dd>
+                            <dt>Created on</dt>
+                            <dd class="o_Activity_detailsCreation">
+                                <t t-esc="formattedCreateDatetime"/>
+                                by
+                                <img t-att-src="creatorAvatarSource" height="18" width="18"
+                                    class="o_object_fit_cover rounded-circle mr4"
```

Comment:
avoid weird style-only meaningful classnames like `o_object_fit_cover ` or `mr4` and prefer setting styler ules on this part `.o_Activity_creatorAvatarImage`.